elixir/sum-of-multiples/lib/sum_of_multiples.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SumOfMultiples do
+  @doc """
+  Adds up all numbers from 1 to a given end number that are multiples of the factors provided.
+  """
+
+
+  def to(limit, factors) do
+    Enum.reduce(0..limit-1, 0, & &2 + get_factor_sum(&1, factors))
+
+
```

Comment:
:broom: Might want to clean up those blank lines?

Reply:
done ! :)